### ISSUE TITLE
Utilize source map data in Testem's PhantomJS stack traces and logs

### ISSUE THREAD

**Issue by brettz9** [alekseykulikov/browserify-test#8]
While I reported my main concern already at https://github.com/testem/testem/issues/1095 , this TDD issue is I think particularly critical with `browserify-test` as we always assume a bundle so I thought you might be interested in tracking it here as well. Note that passing `debug: true` with `browserifyOptions` is not currently sufficient as PhantomJS simply will not use the source map info.

My limited testing also seems to suggest that Safari (at least 10.0.3) will only preserve source maps if we change your monkey-patch of `Server.prototype.serveHomePage` to write the user's browserified bundle buffer to an external file. As mentioned in the referenced issue, even here in Safari (as opposed to PhantomJS), it doesn't list the source line numbers in test error reporting, but it at least lists the source maps in console. I'd also therefore like to know whether you'd be open to a PR to preserve those source maps?

**Comment by alekseykulikov** [comment]
I agree, it would be nice to have proper source maps support, looking forward for PR.
